cnxarchive/views.py

```diff
@@ -428,6 +428,41 @@ def _get_licenses(cursor):
     return [json_row[0] for json_row in cursor.fetchall()]
 
 
+def get_query_vars(query_string):
+    qlist = query_string.split("&")
+    qdict = {}
+    for param in qlist:
+        param_split = param.split("=")
+        if len(param_split) == 2:
+            qdict[param_split[0]] = param_split[1]
+    return qdict
+
+
+def format_author(personids, settings):
+    """
+    Takes a list of personid's and searches in the persons table to get their
+    full names and returns a list of the full names as a string.
+    """
+    personids_string = "("
+    for i in range(len(personids) - 1):
+        personids_string += "'{},'".format(peronids[i])
```

Comment:
Also, the spacer there is `"'{},'"` it should be `"'{}',"`. Otherwise you end up with `['id1,''id2']` rather than `['id1','id2']`